Sts1CobcSw/FileSystem/LfsWrapper.ipp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <Sts1CobcSw/FileSystem/LfsWrapper.hpp>
+
+#include "Sts1CobcSw/FileSystem/ErrorsAndResult.hpp"
```

Comment:
Only the corresponding .hpp file may be included in an .ipp file, so this line must be removed.

Reply:
Fixed in commit https://github.com/SpaceTeam/STS1_COBC_SW/pull/287/commits/e6efcca7d93be7354637641ebe498b194f0e4f44